controller/src/beerocks/master/beerocks_master_mrouter.cpp

```diff
@@ -55,7 +55,9 @@ bool master_mrouter::init()
         ieee1905_1::eMessageType::AP_AUTOCONFIGURATION_WSC_MESSAGE,
         ieee1905_1::eMessageType::CHANNEL_PREFERENCE_REPORT_MESSAGE,
         ieee1905_1::eMessageType::CHANNEL_SELECTION_RESPONSE_MESSAGE,
+        ieee1905_1::eMessageType::CLIENT_STEERING_BTM_REPORT_MESSAGE,
```

Comment:
Again, contrary to @morantr, I think the controller and the agent part should be in separate commits.